docs/learn-papers.md

```diff
@@ -0,0 +1,36 @@
+---
+id: learn-papers
+title: Polkadot Papers
+sidebar_label: Polkadot Papers
+---
+
+## Polkadot's White Paper
+
+Polkadot's original white paper is a technical summary around one possible direction of implementing
+the Polkadot network. This paper uses rationale and technical details to support why this direction
+is beneficial. This original white paper also explains how each core component in Polkadot works
+together to build this decentralized network. Polkadot's white paper can be found
+[here](https://polkadot.network/PolkaDotPaper.pdf).
```

Comment:
Consider removing this, and linking to it in the beginning of the paragraph like you did with the overview paper. Two reasons - one is consistency, and the other is accessibility. When screen readers go through text, they just read "here" to the people when listing links, unless the link also has a title which in this case it does not. Having the link be something like `[Polkadot's white paper](url)` makes the link more accessible.